core/src/main/java/org/apache/iceberg/inmemory/InMemoryCatalog.java

```diff
@@ -337,4 +394,67 @@ protected String tableName() {
       return tableIdentifier.toString();
     }
   }
+
+  private class InMemoryViewOperations extends BaseViewOperations {
+    private final FileIO fileIO;
```

Comment:
Minor: prefer `io` to `fileIO`. Similar to my other comment, it isn't necessary to duplicate the information from the type.

Reply:
updated